cfme/cloud/instance/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 from navmazing import NavigateToSibling, NavigateToAttribute
 from selenium.common.exceptions import NoSuchElementException
+from fixtures.pytest_store import store
```

Comment:
See utils.appliance.get_or_create_current_appliance(), I think that may be more appropriate to use here.